pkg/discovery/servicehandler.go

```diff
@@ -37,50 +38,41 @@ type ServiceHandler interface {
 }
 
 type serviceHandler struct {
-	stage               string
+	muleEnv             string
```

Comment:
the "stage" field was incorrect. This is actually the Mulesoft environment we are connected to. The "stage" will come from the API product version.